src/app/GitUI/CommandsDialogs/RevisionDiffControl.Designer.cs

```diff
@@ -111,57 +112,18 @@ private void InitializeComponent()
             // 
-            // BlameControl
-            // 
-            BlameControl.Dock = DockStyle.Fill;
-            BlameControl.Location = new Point(0, 0);
-            BlameControl.Margin = new Padding(0);
-            BlameControl.Name = "BlameControl";
-            BlameControl.Size = new Size(300, 360);
-            BlameControl.TabIndex = 1;
-            // 
             // DiffContextMenu
             // 
-            DiffContextMenu.Items.AddRange(new ToolStripItem[] {
-            diffUpdateSubmoduleMenuItem,
-            diffResetSubmoduleChanges,
-            diffStashSubmoduleChangesToolStripMenuItem,
-            diffCommitSubmoduleChanges,
-            submoduleStripSeparator,
-            stageFileToolStripMenuItem,
-            unstageFileToolStripMenuItem,
-            resetFileToToolStripMenuItem,
-            cherryPickSelectedDiffFileToolStripMenuItem,
-            toolStripSeparator32,
-            openWithDifftoolToolStripMenuItem,
-            diffOpenWorkingDirectoryFileWithToolStripMenuItem,
-            diffOpenRevisionFileToolStripMenuItem,
-            diffOpenRevisionFileWithToolStripMenuItem,
-            saveAsToolStripMenuItem1,
-            diffEditWorkingDirectoryFileToolStripMenuItem,
-            diffDeleteFileToolStripMenuItem,
-            diffToolStripSeparator13,
-            copyPathsToolStripMenuItem,
-            openContainingFolderToolStripMenuItem,
-            toolStripSeparator33,
-            diffShowInFileTreeToolStripMenuItem,
-            diffFilterFileInGridToolStripMenuItem,
-            fileHistoryDiffToolstripMenuItem,
-            blameToolStripMenuItem,
-            findInDiffToolStripMenuItem,
-            showFindInCommitFilesGitGrepDialogToolStripMenuItem,
-            showFindInCommitFilesGitGrepToolStripMenuItem,
-            toolStripSeparatorScript,
-            runScriptToolStripMenuItem});
+            DiffContextMenu.Items.AddRange(new ToolStripItem[] { diffUpdateSubmoduleMenuItem, diffResetSubmoduleChanges, diffStashSubmoduleChangesToolStripMenuItem, diffCommitSubmoduleChanges, submoduleStripSeparator, stageFileToolStripMenuItem, unstageFileToolStripMenuItem, resetFileToToolStripMenuItem, cherryPickSelectedDiffFileToolStripMenuItem, toolStripSeparator32, openWithDifftoolToolStripMenuItem, diffOpenWorkingDirectoryFileWithToolStripMenuItem, diffOpenRevisionFileToolStripMenuItem, diffOpenRevisionFileWithToolStripMenuItem, saveAsToolStripMenuItem1, diffEditWorkingDirectoryFileToolStripMenuItem, diffDeleteFileToolStripMenuItem, diffToolStripSeparator13, copyPathsToolStripMenuItem, openContainingFolderToolStripMenuItem, toolStripSeparator33, diffShowInFileTreeToolStripMenuItem, diffFilterFileInGridToolStripMenuItem, fileHistoryDiffToolstripMenuItem, blameToolStripMenuItem, findInDiffToolStripMenuItem, showFindInCommitFilesGitGrepDialogToolStripMenuItem, showFindInCommitFilesGitGrepToolStripMenuItem, toolStripSeparatorScript, runScriptToolStripMenuItem });
```

Comment:
Why is the format as this?
I can use difftastic to see that the order is the same but it seem unneeded.

Otherwise no comment on the changes

Reply:
> Why is the format as this?

As mentioned in the commit message "chore(RevisionDiffControl): Saved without changes using Designer 17.11.3".
Don't ask me!

Reply:
I prefer to revert this change

Reply:
reverted because there are is another nonsense change by the Designer to be reverted.

Any further objections / vetos, @gitextensions/git-extensions-team?

Reply:
merge